src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/CountDocumentsCommands.java

```diff
@@ -0,0 +1,21 @@
+package io.stargate.sgv2.jsonapi.api.model.command.impl;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import io.stargate.sgv2.jsonapi.api.model.command.Filterable;
+import io.stargate.sgv2.jsonapi.api.model.command.ReadCommand;
+import io.stargate.sgv2.jsonapi.api.model.command.clause.filter.FilterClause;
+import javax.annotation.Nullable;
+import javax.validation.Valid;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+@Schema(
+    description =
+        "Command that returns count of documents in a collection based on the collection.")
+@JsonTypeName("countDocuments")
+public record CountDocumentsCommands(
```

Comment:
Is plural intentional? I assume should this be `CountDocumentsCommand`.

Reply:
mongo command name is countDocuments, so key the class name too in plural